### PR TITLE
Add missing logic to SZVAV fan coil

### PR DESCRIPTION
**Pull request by rraustad**
- DoNoPublish: This pull request includes changes that shouldn't be included in the change log
- No diff's expected
- Add's missing logic found when creating unit test

### CODE REVIEW

File: src/EnergyPlus/FanCoilUnits.cc [hunk 1]
```diff
@@ -2312,6 +2312,28 @@ namespace FanCoilUnits {
 						SetComponentFlowRate( Node( FanCoil( FanCoilNum ).ColdControlNode ).MassFlowRate, FanCoil( FanCoilNum ).ColdControlNode, FanCoil( FanCoilNum ).ColdPlantOutletNode, FanCoil( FanCoilNum ).CWLoopNum, FanCoil( FanCoilNum ).CWLoopSide, FanCoil( FanCoilNum ).CWBranchNum, FanCoil( FanCoilNum ).CWCompNum );
 					}
 
+                // operate at variable coil water flow rate to meet load
+                } else if ( QUnitOutMax < 0.0 && ( QZnReq > QUnitOutMax ) ) {
```
Comment (rraustad) [inline src/EnergyPlus/FanCoilUnits.cc]: Missed this on original develop.

File: tst/EnergyPlus/unit/FanCoilUnits.unit.cc [hunk 2]
```diff
@@ -1054,44 +1054,53 @@ namespace EnergyPlus {
...
+		EXPECT_GT( Node( 1 ).MassFlowRate, FanCoil( 1 ).MaxAirMassFlow * FanCoil( 1 ).LowSpeedRatio );
+		EXPECT_LT( Node( 1 ).MassFlowRate, FanCoil( 1 ).MaxAirMassFlow );
+
+		// expect full flow and meet capacity
+		ZoneSysEnergyDemand( 1 ).RemainingOutputReqToHeatSP = 0.0;
+		ZoneSysEnergyDemand( 1 ).RemainingOutputReqToCoolSP = -4000.0;
+		QZnReq = -4000.0;
+		Sim4PipeFanCoil( FanCoilNum, ZoneNum, ZoneNum, FirstHVACIteration, QUnitOut, QLatOut );
+		EXPECT_NEAR( QZnReq, QUnitOut, 5.0 );
+		EXPECT_NEAR( Node( 1 ).MassFlowRate, FanCoil( 1 ).MaxAirMassFlow, 0.0000000001 );
```
Comment (rraustad) [inline tst/EnergyPlus/unit/FanCoilUnits.unit.cc:1080]: New variation of unit test now passes.

File: tst/EnergyPlus/unit/FanCoilUnits.unit.cc [hunk 3]
```diff
@@ -1054,44 +1054,53 @@ namespace EnergyPlus {
 
 		// expect full flow and meet capacity
 		EXPECT_NEAR( QZnReq, QUnitOut, 5.0 );
-		EXPECT_NEAR( Node( 1 ).MassFlowRate, FanCoil( 1 ).FanAirVolFlow, 0.0000000001 );
+		EXPECT_NEAR( Node( 1 ).MassFlowRate, FanCoil( 1 ).MaxAirMassFlow, 0.0000000001 );
```
Comment (rraustad) [inline tst/EnergyPlus/unit/FanCoilUnits.unit.cc:1057]: Coincidentally, FanAirVolFlow and FC MaxAirMassFlow were the same in this unit test. Corrected variable so the comparison doesn't look suspect.

**Comment by Myoldmopar** [comment]
@rraustad I don't have any issues with the changes, except that something went awry with the indentation in the source file.  It looks like the editor used spaces instead of tabs.  Can you clean that up?  I'll merge it in after that.

**Comment by rraustad** [comment]
Done?

**Pull request merged**